index.html

```diff
@@ -7,8 +7,10 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <link rel="stylesheet" href="styles.css">
         <link rel="shortcut icon" type="image/png" href="/approachable-open-source-logo.png"/>
+        <link rel="alternate" href="https://github.com" hreflang="en-us" />
     </head>
     <body>
+        <a href="#main-content" class="skip-link">Skip to Main Content</a> 
         <div class="text-container">
```

Comment:
We should find a way to target this element so that when the skip link is followed it directs the user to this content. Reference #15 for more info.

Reply:
I would be ok with adding `id="main-content"` to the tag. Will that do it? @MobGG, since it is your Pull Request, do you want to tackle that?